tests/test_api_skeleton.py

```diff
@@ -297,7 +314,7 @@ def test_get_package_metadata(testing_config, mock_metadata, pylint_metadata):
         mock_metadata,
         {},
         ".",
-        "3.7",
```

Comment:
This is probably the thing that fixed the CI.
Python >=3.8 handles `TemporaryDirectory` cleanups better: https://github.com/python/cpython/pull/10320 .

Reply:
hm interesting